course_catalog/etl/xpro.py

```diff
@@ -165,3 +186,41 @@ def transform_programs(programs):
         }
         for program in programs
     ]
+
+
+def transform_content_files(course_tarpath):
+    """
+    Pass content to tika, then return a JSON document with the transformed content inside it
+
+    Args:
+        course_tarpath (str): The path to the tarball which contains the OLX
+    """
+    content = []
+    with TemporaryDirectory() as inner_tempdir:
+        check_call(["tar", "xf", course_tarpath], cwd=inner_tempdir)
+        olx_path = glob.glob(inner_tempdir + "/*")[0]
+        for document, key in documents_from_olx(olx_path):
+            tika_output = extract_text_metadata(document).get("content") or ""
+            content.append({"content": tika_output.strip(), "key": key})
+    return content
```

Comment:
Is there any info that could be used for fields like 'url', 'title', 'description', etc?  The `ContentFile` objects being created for xPro are pretty sparse, with just 'content' and 'key' populated.  

Also, unless always blank, should populate the additional `content_` fields from tika as for ocw at https://github.com/mitodl/open-discussions/blob/d975f790c3b5459a8fe7a94bebbde5dfbfc9dec6/course_catalog/etl/ocw.py#L131-L139